.travis.yml

```diff
@@ -70,3 +70,6 @@ matrix:
     - if: type != cron
       compiler: "clang"
       env: CI_BUILD_TARGET="sitltest-rover"
+    - if: type != cron
+      compiler: "clang"
+      env: CI_BUILD_TARGET="sitltest-sub"
```

Comment:
This makes a new job, we don't want that, we want to run this on the same job Rover autotest does:
```
env: CI_BUILD_TARGET="sitltest-rover sitltest-sub"
```